int32 lane_number
string lane_direction
int32[] x
int32[] y
